alcs-api/src/providers/typeorm/migrations/1659047420726-seed_application_types.ts

```diff
@@ -4,15 +4,15 @@ export class seedApplicationTypes1659047420726 implements MigrationInterface {
-      ('1a8d99d2-1601-4af2-afb1-f5429b32be5a', NULL, now(), NULL, 'alcs-api', NULL, 'POFO', 'Placement of Fill Only', 'Placement of Fill', 'Fill', '#b2ff59', '#000'),
-      ('71dfa994-5667-49a6-9be4-1822b45ac7a3', NULL, now(), NULL, 'alcs-api', NULL, 'ROSO', 'Removal of Soil Only', 'Removal of Soil', 'Rem', '#ff9800', '#000'),
-      ('7f99b32b-d1ea-4147-bf9c-4a81fbbac6cb', NULL, now(), NULL, 'alcs-api', NULL, 'PFRS', 'Placement of Fill and Removal of Soil', 'Placement of Fill/Removal of Soil', 'Fill + Rem', '#ffeb3b', '#000'),
-      ('8a21179e-6fa0-4584-b4f1-d72c3f758b0a', NULL, now(), NULL, 'alcs-api', NULL, 'INCL', 'Inclusion Permit', 'Inclusion', 'Inclu', '#ff859f', '#000'),
-      ('9f444184-09be-4a6e-8095-c702cc4681c7', NULL, now(), NULL, 'alcs-api', NULL, 'SUBD', 'Subdivision Permit', 'Subdivison', 'Subdiv', '#c12bdb', '#fff'),
-      ('dc07b1e7-2f58-43ab-817f-7ef83d9d35b0', NULL, now(), NULL, 'alcs-api', NULL, 'EXCL', 'Exclusion Permit', 'Exclusion', 'Exclus', '#45f4f4', '#000'),
-      ('e272645d-dcd2-405b-8caf-f94a17500f14', NULL, now(), NULL, 'alcs-api', NULL, 'TUTP', 'Transportation, Utility, Trail Permits', 'Transportation/Utility/Trail', 'T/U/T', '#0a1e8d', '#fff'),
+      ('1a8d99d2-1601-4af2-afb1-f5429b32be5a', NULL, now(), NULL, 'alcs-api', NULL, 'POFO', 'Placement of Fill Only', 'Placement of Fill', 'SOIL', '#b2ff59', '#000'),
+      ('71dfa994-5667-49a6-9be4-1822b45ac7a3', NULL, now(), NULL, 'alcs-api', NULL, 'ROSO', 'Removal of Soil Only', 'Removal of Soil', 'SOIL', '#b2ff59', '#000'),
+      ('7f99b32b-d1ea-4147-bf9c-4a81fbbac6cb', NULL, now(), NULL, 'alcs-api', NULL, 'PFRS', 'Placement of Fill and Removal of Soil', 'Placement of Fill/Removal of Soil', 'SOIL', '#b2ff59', '#000'),
+      ('8a21179e-6fa0-4584-b4f1-d72c3f758b0a', NULL, now(), NULL, 'alcs-api', NULL, 'INCL', 'Inclusion Permit', 'Inclusion', 'INC', '#ff859f', '#000'),
+      ('9f444184-09be-4a6e-8095-c702cc4681c7', NULL, now(), NULL, 'alcs-api', NULL, 'SUBD', 'Subdivision Permit', 'Subdivison', 'SD', '#c12bdb', '#fff'),
+      ('dc07b1e7-2f58-43ab-817f-7ef83d9d35b0', NULL, now(), NULL, 'alcs-api', NULL, 'EXCL', 'Exclusion Permit', 'Exclusion', 'EXC', '#45f4f4', '#000'),
+      ('e272645d-dcd2-405b-8caf-f94a17500f14', NULL, now(), NULL, 'alcs-api', NULL, 'TURP', 'Transportation, Utility, Trail Permits', 'Transportation, Utility, or Recreational Trail', 'TUR', '#0a1e8d', '#fff'),
       ('ee666f9f-4d94-44dd-b081-535c7aeb50c8', NULL, now(), NULL, 'alcs-api', NULL, 'NFUP', 'Non-Farm Use Permit', 'Non-Farm Use', 'NFU', '#228820', '#fff'),
-      ('f985b157-cc10-4475-8504-06279a2cd73d', NULL, now(), NULL, 'alcs-api', NULL, 'NARS', 'Non-Adhering Residential Use', 'Non-Adhering Residential Use', 'NA Res Use', '#454545', '#fff');
+      ('f985b157-cc10-4475-8504-06279a2cd73d', NULL, now(), NULL, 'alcs-api', NULL, 'NARU', 'Non-Adhering Residential Use', 'Non-Adhering Residential Use', 'NARU', '#454545', '#fff');
```

Comment:
I'm wondering if we want to use 'ON CONFLICT'? so this will be safe to run multiple times?

Reply:
I'm not sure to be honest. Do you want me to change it?

Reply:
I think if we end up updating this further then it is not a bad idea. Let's hold of till we get deployed db and starting from that point we will right migrations in safe mode